storage-mirror-firestore/POSTINSTALL.md

```diff
@@ -58,11 +58,12 @@ const itemsSnapshot = await firestore
 
 ### (Optional) Backfill
 
-If you already have objects in GCS that should be mirrored to Firestore, you can use the backfill tool in the stress_test subdirectory. If you application default credentials are set up, you can just run `npm run-script start -- backfill` from that directory.
+If you already have objects in GCS that should be mirrored to Firestore, you can use the backfill tool in the stress_test subdirectory. If your application default credentials are set up, you can just run `npm run-script start -- backfill` from that directory.
 
 ### (Optional) Cleaning Up Tombstones
 
-*TODO*: Implement clean-up tool for tombstone documents
+If you want to cleanup tombstone records, you can use the cleanup tool in the stress_test subdirectory. If your application default credentials are set up, you can just run `npm run-script start -- clean-tombstones` from that directory. You can optionally also specify the
```

Comment:
Developers will be reading this in the Firebase console, so please link to the directory. We can also consider publishing to the [@firebaseextensions npm org](https://www.npmjs.com/org/firebaseextensions).

Reply:
I know it is mentioned in the preinstall, but consider adding a line at the start of this explaining what a tombstone is

Reply:
Would you like us to update the backfill script docs too since this was a copy paste of that one with minor tweaks?

Reply:
Good point. Per other comment, let's save major postinstall changes for #59